content/version/2/0/code_of_conduct.fr.md

```diff
@@ -0,0 +1,132 @@
+# Charte Code de Conduite Contributeurs
+
+## Notre engagement
+
+En tant que membres, contributeurs et dirigeants, nous nous 
+engageons à rendre la participation dans notre communauté 
+une expérience sans harcelement pour tous, quel que soit l'âge, 
+la physionomie, la présence d'un handicap visible ou invisible, l'ethnie, 

```

Comment:
```suggestion
la taille corporelle, le handicap visible ou invisible, l'appartenance ethnique,
```

"physionomie" refers more to the face than to body size, imho.

Reply:
I think I did something bad, the suggestion is now marked as outdated, so I can't commit it.
Maybe we should open a new one ? I don't know what's the good way to fix it.

Reply:
Yeah, the UI is a bit confusing here.